xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -934,8 +931,8 @@ void CPVRClients::ProcessMenuHooks(int iClientID, PVR_MENUHOOK_CAT cat, const CF
       if (selection >= 0)
       {
         itrClients = clients.begin();
-        for (int i = 0; i < selection; i++)
-          itrClients++;
+        for (int i = 0; i < selection; ++i)
+          ++itrClients;
         iClientID = itrClients->first;
```

Comment:
Was also not sure now, but also with C11 not possible. It still use the `key_type` for `at` :(.
Has looked here: http://www.cplusplus.com/reference/map/map/at/